res/supersonic-desktop.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+Name=Supersonic
+Comment=A lightweight cross-platform desktop client for Subsonic music servers
+Path=/usr/bin
+Exec=supersonic-desktop
+Terminal=false
+Icon=supersonic-desktop
```

Comment:
Are you using the `res/appicon-500.png` file as the icon? I'm confused as how the icon path can be the same as the exec path

Reply:
Ah, I see that this isn't supposed to be a full icon path, it's relative to icon search paths. But the file name should still end in .png, no?

Reply:
The system looks for the icons ending with .jpg .png .ico among others. As per [the free desktop standard](https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html) it looks in these successive locations, until the first one is found.

```
$HOME/.icons
$XDG_DATA_DIRS/icons
/usr/share/pixmaps
```

It's not required to name the extension.